guidelines/sc/21/character-key-shortcuts.html

```diff
@@ -14,7 +14,7 @@ <h4>Character Key Shortcuts</h4>
     <dd>A <a>mechanism</a> is available to turn the shortcut off;</dd>
       						
    <dt>Remap</dt>
-     <dd>A mechanism is available to remap the shortcut to use one or more non-printable keyboard characters (e.g. Ctrl, Alt, etc);</dd>
+     <dd>A mechanism is available to remap the shortcut to include one or more non-printable keyboard keys (e.g., Ctrl, Alt);</dd>
```

Comment:
Agree on the e.g./etc.

I think that use/include an be explained in understanding and probably needs to be clarified whether include or use is the term used/included in the SC text.